acvm-repo/acir_field/src/generic_ark.rs

```diff
@@ -70,6 +70,10 @@ pub trait AcirField:
 
     fn to_hex(self) -> String;
 
+    /// Returns the value of this field as a hex string with leading zeroes removed.
+    /// A singular '0' will be prepended as well if the trimmed string has an odd length.
+    fn to_short_hex(self) -> String;
```

Comment:
Maybe it would be worth highlighting the difference between `to_hex` and `to_short_hex` here?

Reply:
Done! It's now a bit strange that one of them has the `0x` prefix and one doesn't, but at least it's documented. This should all be implemented with `Display` to `LowerHex`/`UpperHex` and formatting flags, but it's probably not worth it right now.